assets/src/edit-story/components/checklist/designChecks.js

```diff
@@ -13,29 +13,53 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
 /**
  * Internal dependencies
  */
+import { __ } from '@web-stories-wp/i18n';
```

Comment:
Nit: should be under "External dependencies" above

Reply:
(had to fix a test so this is fixed too)